tests/www/test_security.py

```diff
@@ -969,3 +971,16 @@ def test_users_can_be_found(app, security_manager, session, caplog):
     assert len(users) == 1
     delete_user(app, "Test")
     assert "Error adding new user to database" in caplog.text
+
+
+def test_access_denied_message():
+    with mock.patch.dict(
+        os.environ,
+        {"AIRFLOW__WEBSERVER__ACCESS_DENIED_MESSAGE": "My custom access denied message"},
+        clear=True,
+    ):
+        initialize_config()
+        assert conf.get("webserver", "access_denied_message") == "My custom access denied message"
+
+    initialize_config()
+    assert conf.get("webserver", "access_denied_message") == "Access is Denied"
```

Comment:
could you replace the `conf.get("webserver", "access_denied_message")` by `get_access_denied_message` method?
And maybe separate them in two tests: 
- test_default_access_denied_message
- test_custom_access_denied_message

Reply:
Yeah why not. Very valid suggestions. Incorporated them.